smartsim/log.py

```diff
@@ -77,15 +77,14 @@ def _get_log_level() -> str:
     :returns: Log level for coloredlogs
     :rtype: str
     """
-    log_level = os.environ.get("SMARTSIM_LOG_LEVEL", "info").lower()
-    if log_level == "quiet":
+    if user_log_level == "quiet":
```

Comment:
can we shorten this to:

```py
if user_log_level in ['quiet', 'info', 'debug','warning']:
    return user_log_level

if user_log_level == "developer":
    return "info"
```

Reply:
I think you missed one case, but I got the point and modified implementation accordingly.